src/@next/IndexTable/IndexTableStyle.ts

```diff
@@ -0,0 +1,1745 @@
+      0 0.0625rem 0.1875rem 0 rgba(63, 63, 68, 0.15);
+    --p-shadow-deep: 0 0 0 0.0625rem rgba(6, 44, 82, 0.1),
+      0 0.125rem 1rem rgba(33, 43, 54, 0.08);
+    --p-shadow-button: 0 0.0625rem 0 rgba(0, 0, 0, 0.05);
+    --p-shadow-popover: 0 0.1875rem 0.375rem -0.1875rem rgba(23, 24, 24, 0.08),
+      0 0.5rem 1.25rem -0.25rem rgba(23, 24, 24, 0.12);
+    --p-duration-100: 100ms;
+    --p-border-radius-1: 0.25rem;
+    --p-border-radius-2: 0.5rem;
+    --p-border-width-1: 0.0625rem;
+    --p-border-divider: var(--p-border-width-1) solid var(--p-divider);
+    --p-space-025: 0.0625rem;
+    --p-space-05: 0.125rem;
+    --p-space-1: 0.25rem;
+    --p-space-2: 0.5rem;
+    --p-space-3: 0.75rem;
+    --p-space-4: 1rem;
+    --p-space-5: 1.25rem;
+  }
+
+  .Polaris-Text--root {
+    margin: 0;
+    text-align: inherit;
+  }
+
+  .Polaris-Text--visuallyHidden {
+    position: absolute !important;
+    top: 0;
+    width: 0.0625rem !important;
+    height: 0.0625rem !important;
+    margin: 0 !important;
+    padding: 0 !important;
+    overflow: hidden !important;
+    clip-path: inset(50%) !important;
+    border: 0 !important;
+    white-space: nowrap !important;
+  }
+
+  .Polaris-Text--regular {
+    font-weight: 400;
+  }
+
+  .Polaris-Text--bodySm {
+    font-size: 0.75rem;
+    line-height: 1rem;
+  }
+
+  .Polaris-Icon {
+    display: block;
+    height: 1.25rem;
+    width: 1.25rem;
+    max-height: 100%;
+    max-width: 100%;
+    margin: auto;
+  }
+
+  .Polaris-Icon__Svg {
+    position: relative;
+    display: block;
+    width: 100%;
+    max-width: 100%;
+    max-height: 100%;
+  }
+
+  .Polaris-Scrollable {
+    --pc-scrollable-shadow-size: var(--p-space-5);
+    --pc-scrollable-shadow-bottom: inset 0
+      calc(var(--pc-scrollable-shadow-size) * -1)
+      var(--pc-scrollable-shadow-size)
+      calc(var(--pc-scrollable-shadow-size) * -1)
+      var(--p-hint-from-direct-light);
+    --pc-scrollable-shadow-top: inset 0 var(--pc-scrollable-shadow-size)
+      var(--pc-scrollable-shadow-size)
+      calc(var(--pc-scrollable-shadow-size) * -1)
+      var(--p-hint-from-direct-light);
+    --pc-scrollable-max-height: none;
+    -webkit-overflow-scrolling: touch;
+    position: relative;
+    max-height: var(--pc-scrollable-max-height);
+    overflow-x: hidden;
+    overflow-y: hidden;
+  }
+
+  .Polaris-Scrollable:focus {
+    outline: 0.125rem solid var(--p-focused);
+    outline-offset: var(--p-space-05);
+  }
+
+  .Polaris-Scrollable--horizontal {
+    overflow-x: auto;
+  }
+
+  .Polaris-Scrollable--vertical {
+    overflow-y: auto;
+  }
+
+  .Polaris-Scrollable--hasTopShadow {
+    box-shadow: var(--pc-scrollable-shadow-top);
+  }
+
+  .Polaris-Scrollable--hasBottomShadow {
+    box-shadow: var(--pc-scrollable-shadow-bottom);
+  }
+
+  .Polaris-Scrollable--hasTopShadow.Polaris-Scrollable--hasBottomShadow {
+    box-shadow: var(--pc-scrollable-shadow-top),
+      var(--pc-scrollable-shadow-bottom);
+  }
+
+  .Polaris-Popover {
```

Comment:
Do we still need Popover styles here, as they have been added in PopoverStyle?

Reply:
hmm.. seems this one is breaking 🤔 
![Screenshot 2023-02-03 at 4 42 13 PM](https://user-images.githubusercontent.com/7514754/216553599-bdace342-d4b1-46d8-8ae7-c12754614bc8.png)